genai_stack/constants/llm_cache.py

```diff
@@ -0,0 +1,11 @@
+LLM_CACHE_MODULE = "genai_stack.llm_cache"
+LLM_CACHE_CONFIG_KEY = "llm_cache"
+
+
+class LLM_Cache:
+    LANGCHAIN = "cache"
```

Comment:
We can call this default since we are implementing our own caching and not using langchain

Reply:
We just changed this for LLM Cache but there are other components where were have used langchain in the name which cannot be changes since there are use cases and implementation in other places.